libcloudforensics/providers/gcp/internal/compute_resources.py

```diff
@@ -12,7 +12,7 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""Google compute functionality."""
+"""Google compute resources."""
```

Comment:
the product name is Compute Engine - we should probably be using the full name here